src/Cake.Web/App_Data/docs/fundamentals/configuration.md

```diff
@@ -0,0 +1,34 @@
+---
+content-type: markdown
+---
+
+Cake supports the concept of external configuration, to allow the internals of how Cake operates to be controlled from either:
```

Comment:
Might be good to point out that the different methods have different priorities.